test/unit/subscription_mailer_test.rb

```diff
@@ -61,4 +61,23 @@ class SubscriptionMailerTest < ActionMailer::TestCase
     assert_equal "[PublicLab] #{user.username} liked your question", email.subject
     assert email.body.include?("Public Lab contributor #{user.username} (https://#{request_host}/profile/#{user.username}) just liked your question")
   end
+
+  test 'notify users who do not follow particular tag' do
```

Comment:
Can we be even more descriptive here? 'notify users who follow a newly added tag but were not previously notified based on the nodes existing tags'